### PR TITLE
PR: Update README's sponsors section

### PR DESCRIPTION
**Pull request by dalthviz**
## Description of Changes

* [ ] Wrote at least one-line docstrings (for any new functions)
* [ ] Added unit test(s) covering the changes (if testable)
* [ ] Included a screenshot (if affecting the UI)

[A preview](https://github.com/dalthviz/spyder/blob/sponsors/README.md)

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct: dalthviz

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -258,18 +258,12 @@ a Python version greater than 2.7 or 3.4 (Python <=3.3 is no longer supported).
...
-## Backers
-
-Support us with a monthly donation and help us continue our activities.
-
-[![Backers](https://opencollective.com/spyder/backers.svg)](https://opencollective.com/spyder#support)
-
-
 ## Sponsors
 
 Become a sponsor to get your logo on our README on Github.
```
Comment (ccordoba12) [inline README.md]: Please use here the same text as in the webpage:

> Spyder is funded thanks to ...

File: README.md [hunk 2]
```diff
@@ -258,18 +258,12 @@ a Python version greater than 2.7 or 3.4 (Python <=3.3 is no longer supported).
...
 
+[![Quansight](https://static.wixstatic.com/media/095d2c_2508c560e87d436ea00357abc404cf1d~mv2.png/v1/crop/x_0,y_9,w_915,h_329/fill/w_191,h_66,al_c,usm_0.66_1.00_0.01/095d2c_2508c560e87d436ea00357abc404cf1d~mv2.png)](https://www.quansight.com/) [![Numfocus](https://numfocus.org/wp-content/uploads/2018/01/optNumFocus_LRG.png)](https://numfocus.org/)
```
Comment (ccordoba12) [inline README.md]: Please make Quansight and Numfocus bigger than the Open Collective image below (about the same size you use in the webpage).

File: README.md (hunk 1 above)
Comment (ccordoba12) [reply README.md]: And remove this text here.

File: README.md (hunk 2 above)
Comment (dalthviz) [reply README.md]: imagen

Comment (ccordoba12) [reply README.md]: Please make the Quansight and Numfocs logos even bigger (they have to look bigger than the Open collective image) and add the rest of the text we added in the webpage:

> and the donations we have received ...

----

So the idea is to change this section to be as close as possible as the Sponsors section you added to our webpage.

Comment (dalthviz) [reply README.md]: imagen

**Pull request merged**